editor/editor_node.cpp

```diff
@@ -4391,17 +4391,17 @@ void EditorNode::_project_run_started() {
 		log->clear();
 	}
 
-	if (bool(EDITOR_GET("run/output/always_open_output_on_play"))) {
+	int action_on_play = EDITOR_GET("run/bottom_panel/action_on_play");
```

Comment:
I know this was here originally, but I wonder why it has to be done twice? (the other one being in debugger)
This one doesn't even handle the Open Debugger case; maybe it can be removed.

Reply:
Hehe, I asked myself the same thing! I don't known, I figured at the time that there was a reason. I'll remove it.

Reply:
Or you could remove the other one, so that start and stop are handled in the same file. Not sure if the debugger is available here though.